steam_app/FavoritesTableView/FavoritesTableViewCell.swift

```diff
@@ -21,8 +21,8 @@ class FavoritesTableViewCell: UITableViewCell {
         
     }
     
-    func cellConfiguration(cell: FavoritesTableViewCell, indexPath: IndexPath) {
-        let favs = favsList[indexPath.row]
+    func cellConfiguration(cell: FavoritesTableViewCell, indexPath: IndexPath, data: [Favorite]) {
+        let favs = data[indexPath.row]
```

Comment:
I would suggest to give more concrete names here like below:
```Swift
func configureCell(with game: Game) {
}
```

The cell should now know about other objects and also, it does not need to know about the index path as well